indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerConfig.java

```diff
@@ -107,8 +113,11 @@ public void configure(Binder binder)
             new IndexingHadoopModule()
         )
     );
-    jsonMapper = injector.getInstance(ObjectMapper.class);
+    JSON_MAPPER = injector.getInstance(ObjectMapper.class);
     properties = injector.getInstance(Properties.class);
```

Comment:
I think `properties` can be removed- it was only used for the "hack to get druid.processing.bitmap property passed down" which you've now got rid of